### PR TITLE
Ported orcish coas from ck2

### PR DESCRIPTION
**Pull request by MemeWizard**
#### Changelog:
- Ported orcish coas from ck2

### CODE REVIEW

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt [hunk 1]
```diff
@@ -668,6 +668,76 @@ colored_emblem_texture_lists = {
 			100 	= "ce_frostwolf.dds"
```
Comment (zumbak04) [inline common/coat_of_arms/template_lists/colored_emblem_lists.txt:668]: They should be moved up.
<details>
<summary>Click to expand</summary>

image

</details>

**Review by zumbak04 — CHANGES_REQUESTED** [review]

File: common/coat_of_arms/template_lists/colored_emblem_lists.txt (hunk 1 above)
Comment (zumbak04) [reply common/coat_of_arms/template_lists/colored_emblem_lists.txt:668]: Nevermind, wait till my answer.

**Review by zumbak04 — APPROVED** [review]
LGTM!

**Review by Remerod — APPROVED** [review]
It all looked good to me!
Nicely done.

**Comment by MemeWizard** [comment]
@zumbak04 @Remerod thanks for reviews

**Pull request merged**